pilot/pkg/model/telemetry.go

```diff
@@ -111,23 +114,23 @@ func shallowMerge(parent, child *tpb.Telemetry) *tpb.Telemetry {
 	if child == nil {
 		return parent
 	}
-	return shallowMergeTracing(parent, child)
+	merged := parent.DeepCopy()
+	shallowMergeTracing(merged, child)
+	shallowMergeAccessLogs(merged, child)
+	return merged
 }
 
-func shallowMergeTracing(parent, child *tpb.Telemetry) *tpb.Telemetry {
-	if parent.GetTracing() == nil || len(parent.GetTracing()) == 0 {
-		return child
+func shallowMergeTracing(parent, child *tpb.Telemetry) {
+	if len(parent.GetTracing()) == 0 {
+		return
 	}
-	if child.GetTracing() == nil || len(child.GetTracing()) == 0 {
-		return parent
+	if len(child.GetTracing()) == 0 {
+		return
 	}
 
-	merged := parent.DeepCopy()
-	childCopy := child.DeepCopy()
-
 	// only use the first Tracing for now (all that is supported)
-	childTracing := childCopy.Tracing[0]
-	mergedTracing := merged.Tracing[0]
+	childTracing := child.Tracing[0]
+	mergedTracing := parent.Tracing[0]
```

Comment:
Irrelevant to this pr, not sure why the API is a slice but the implementation is not consitent?

Reply:
https://github.com/istio/api/pull/1766/files?file-filters%5B%5D=.go&file-filters%5B%5D=.html&file-filters%5B%5D=.json&file-filters%5B%5D=.proto&file-filters%5B%5D=.py#r657580005

Reply:
We will support multiple, just not in the first implementation since we are trying to get this in 1.11 to get user feedback and the code freeze is in 2 days

Reply:
Also, for Tracing in particular, Envoy currently only supports a single backend at a time. That may change, however.